.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+###### Description
+
+
+###### System Info
+macOS 10.
+Xcode 8.
+
+###### Verification
+- [ ] Have you checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
+- [ ] Have you referenced relating tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
+- [ ] Have you checked your Portfile with `port lint`?
+- [ ] Have you tried existing tests with `sudo port test`?
+- [ ] Have you tried a full install with `sudo port install`?
+- [ ] Have you tested basic functionality of all binary files?
```

Comment:
+1 for the text in brackets.

Such rev-bumping would only be required if their are major changes affecting the API or naming or linking of libs involved, no?